src/conmon.c

```diff
@@ -572,10 +588,7 @@ static gboolean oom_cb_cgroup_v2(int fd, GIOCondition condition, G_GNUC_UNUSED g
 	gboolean ret = G_SOURCE_REMOVE;
 
 	/* Drop the inotify events.  */
-	ssize_t num_read = read(fd, &events, sizeof(events));
-	if (num_read < 0) {
-		pwarn("Failed to read events");
-	}
+	read(fd, &events, sizeof(events));
```

Comment:
is this intentional?

Reply:
ah this was in the crio version and not here. fixing